package.json

```diff
@@ -90,7 +90,6 @@
     "@types/express": "^4.17.1",
     "@types/jest": "^24.0.20",
     "@types/luxon": "^1.21.0",
-    "@types/node": "^11.15.0",
```

Comment:
You need to add this dependency back. Also make sure to fix the associated changes in `package-lock.json` (just revert back to what they were.

The end state is that the `Files changed` tab here shouldn't include either `package.json` or `package-lock.json`